Polygon_mesh_processing/doc/Polygon_mesh_processing/Polygon_mesh_processing.txt

```diff
@@ -342,6 +342,27 @@ iteratively filled, refined and faired to get a faired mesh with no hole.
  Holes in the fork model are filled with triangle patches.
  \cgalFigureEnd
 
+subsection HFPerformance Performance
+
+The hole filling algorithm has a complexity which depends on the
+number of vertices.  While \cgalCite{liepa2003filling} has a running
+time of \f$ O(n^3)\f$ , YY in most cases has running time of \f$ O(n
+\log n)\f$.  We were running `triangulate_refine_and_fair_hole()` for
```

Comment:
You should give some idea on the config + the release of CGAL for which it was mesured.